gwswlib/hydx.py

```diff
@@ -8,6 +8,47 @@ class Hydx:
     def __init__(self):
         self.connection_nodes = []
 
+    def check_import_data(self):
+        self._check_on_unique(
+            self.connection_nodes, "identificatieknooppuntofverbinding", True, "knoop"
```

Comment:
Zeker bij zoiets als `True` is het duidelijker als je `remove_doubles=True` doet.
Dat is een voordeel van keyword arguments, je *kan* daarmee duidelijkheid verschaffen.